src/main/java/frc/team2412/robot/subsystems/ArmSubsystem.java

```diff
@@ -75,13 +75,15 @@ public static class ArmConstants { // To find values later
 		// Constraints
 
 		public static final double MAX_ARM_ANGLE = 118;
-		public static final double MIN_ARM_ANGLE = 6;
+		public static final double MIN_ARM_ANGLE = 0;
```

Comment:
Max/min arm angle were to be potentially used in case we want to dampen velocity towards endpoints of the arm, if we still want them, could be used in the future? I'll change min arm angle back to 6.